src/main/java/org/rust/lang/core/psi/RustTokenElementTypes.java

```diff
@@ -203,6 +203,18 @@
         ANDAND
     );
 
+    @NotNull
+    TokenSet ALL_OPS = TokenSet.create(
```

Comment:
Hmm, [SPACE_AROUND_OPS](https://github.com/intellij-rust/intellij-rust/blob/master/src/main/kotlin/org/rust/ide/formatter/impl/utils.kt#L17) may be redefined as `ALL_OPS \ NO_SPACE_AROUND_OPS`